apps/web-giddh/src/assets/locale/all-features/hi.json

```diff
@@ -151,5 +151,29 @@
 	"support": "सहयोग",
 	"talk_to_sales": "सेल्स से बात करें",
 	"help_doc": "सहायता दस्तावेज़",
-	"schedule_demo": "डेमो शेड्यूल करें"
-}
\ No newline at end of file
+	"schedule_demo": "डेमो शेड्यूल करें",
+
```

Comment:
Ignore this space I will update this file once I will get translated code for this file